pkg/controller/upgradeconfig/upgradeconfig_controller.go

```diff
@@ -226,25 +231,29 @@ func (r *ReconcileUpgradeConfig) upgradeCluster(upgrader cub.ClusterUpgrader, uc
 // checks whether it's time to upgrade based on the spec.upgradeAt timestamp
 func isReadyToUpgrade(upgradeConfig *upgradev1alpha1.UpgradeConfig, metricsClient metrics.Metrics) bool {
 	if !upgradeConfig.Spec.Proceed {
-		log.Info("upgrade cannot be proceed", "proceed", upgradeConfig.Spec.Proceed)
+		log.Info("Upgrade cannot proceed", "proceed", upgradeConfig.Spec.Proceed)
 		return false
 	}
 	upgradeTime, err := time.Parse(time.RFC3339, upgradeConfig.Spec.UpgradeAt)
 	if err != nil {
-		log.Error(err, "failed to parse spec.upgradeAt", upgradeConfig.Spec.UpgradeAt)
+		log.Error(err, "Failed to parse spec.upgradeAt", upgradeConfig.Spec.UpgradeAt)
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Thanks Haoran! Good to know about difference in standards for Error and Info msgs.. Reverted it..